source/opt/replace_desc_array_access_using_var_index.cpp

```diff
@@ -253,8 +253,12 @@ void ReplaceDescArrayAccessUsingVarIndex::ReplaceNonUniformAccessWithSwitchCase(
     Instruction* access_chain_final_user, Instruction* access_chain,
     uint32_t number_of_elements,
     const std::deque<Instruction*>& insts_to_be_cloned) const {
-  // Create merge block and add terminator
   auto* block = context()->get_instr_block(access_chain_final_user);
+  // If instruction does not belong to a block (i.e. in the case of OpDecorate),
+  // no replacement needed
```

Comment:
If instruction  -> If the instruction

no replacement needed -> no replacement is needed.  (Add a period.)

Reply:
Done